distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column_Constraint.enso

```diff
@@ -0,0 +1,26 @@
+from Standard.Base import all
+
+## ADVANCED
+   Specifies additional properties for a column created using
+   `Connection.create_table`.
+
+   The support for the constraints may vary between databases.
+type Column_Constraint
+    ## ADVANCED
+       Specifies a default value for the column.
+
+       Argument:
+       - `sql_expression`: The SQL expression to use as the default value. Note that this is a raw SQL expression, so if you want to set a default to a string, you must include the quotes. The quoting style may depend on the database. Never pass unsanitized data to this parameter.
+
+       ! SQL Injection
+
+         Since `sql_expression` is a raw SQL expression, note that malicious
+         data can cause execution of arbitrary SQL queries. Only use this
+         parameter with trusted data.
+    Default (sql_expression : Text)
```

Comment:
```suggestion
    Default_Value (sql_expression : Text)
```

I feel like `Default` isn't specific enough here.
Can we make it so we build from an Enso value type

Reply:
> Can we make it so we build from an Enso value type

I don't understand the question.

Reply:
I initially wanted to use a `Default_Value` but that would work if its argument was `value:Any`. Now it is an expression, as explained in my earlier comment.

Maybe it should be `Default_Expression`?

Reply:
Yeah, I guess that works - it is a SQL expression that is evaluated for each insert.

Just feels that it is a "default value" albeit specified in SQL.